registration/serializers.py

```diff
@@ -79,6 +79,7 @@ class Meta:
             "is_enrolled",
             "current_class",
             "status",
+            "interactions",
```

Comment:
can we add interactions as a read only field? (so that we can deal with the PUT/POST endpoint in a later ticket)